convert2rhel/checks.py

```diff
@@ -293,7 +295,17 @@ def get_unsupported_kmods(host_kmods, rhel_supported_kmods):
     Ignore certain kmods mentioned in the system configs. These kernel modules moved to kernel core, meaning that the
     functionality is retained and we would be incorrectly saying that the modules are not supported in RHEL."""
     return (
-        host_kmods
-        - rhel_supported_kmods
-        - set(system_info.kmods_to_ignore)
+            host_kmods
+            - rhel_supported_kmods
+            - set(system_info.kmods_to_ignore)
     )
+
+
+def check_repos_are_valid():
+    """Check if all used repositories are valid."""
+    output, ret_code = call_yum_cmd(command="makecache",
```

Comment:
Have you checked that the same command works identically on CentOS 6/7/8? I.e. that the command works not only with dnf but with yum as well.

Reply:
Tested with yum without issue.